### ISSUE TITLE
unable to install

### ISSUE THREAD

**Issue by Gozala** [steelbrain/busy-signal#2]
```
nstalling “busy-signal@1.0.0” failed.Hide output…

npm WARN peerDependencies The peer dependency eslint@^2.4.0 included from eslint-config-airbnb will no
npm WARN peerDependencies longer be automatically installed to fulfill the peerDependency 
npm WARN peerDependencies in npm 3+. Your application will need to depend on it explicitly.
npm ERR! Darwin 15.5.0
npm ERR! argv "/Applications/Atom.app/Contents/Resources/app/apm/bin/node" "/Applications/Atom.app/Contents/Resources/app/apm/node_modules/npm/bin/npm-cli.js" "--globalconfig" "/Users/gozala/.atom/.apm/.apmrc" "--userconfig" "/Users/gozala/.atom/.apmrc" "install" "/private/var/folders/0z/nztt5n4d4kg7b43dg9nv_3fc0000gn/T/d-116329-44377-mz68yl/package.tgz" "--target=0.36.8" "--arch=x64"
npm ERR! node v0.10.40
npm ERR! npm  v2.13.3
npm ERR! code EPEERINVALID

npm ERR! peerinvalid The package eslint@2.2.0 does not satisfy its siblings' peerDependencies requirements!
npm ERR! peerinvalid Peer babel-eslint@5.0.4 wants eslint@<2.3.0
npm ERR! peerinvalid Peer eslint-config-airbnb@6.2.0 wants eslint@^2.4.0

npm ERR! Please include the following file with any support request:
npm ERR!     /private/var/folders/0z/nztt5n4d4kg7b43dg9nv_3fc0000gn/T/apm-install-dir-116329-44377-11c8dk1/npm-debug.log
```

**Comment by steelbrain** [comment]
Oh I am sorry, I put dev deps into deps, fixing now

**Comment by steelbrain** [comment]
Fixed in v1.0.1

**Comment by Gozala** [comment]
Thanks
